packages/react-server-cli/gulpfile.babel.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require("gulp"),
+	babel = require("gulp-babel");
+
+gulp.task("default", () => {
+	return gulp.src("src/**/*.js")
+		.pipe(babel({presets: ["es2015", "react"], plugins: ["transform-runtime"]}))
+		.pipe(gulp.dest("target"));
+});
```

Comment:
Please add an "eslint" task so this will get linted by travis.

If you can think of a clean way to get the one in `packages/react-server/gulpfile.js` factored out, even better. :)

You'll also need to patch the top-level gulpfile to pick up `gulpfile.babel.js` files (it currently looks only for `gulpfile.js`).